pages/api/auth/[...nextauth].ts

```diff
@@ -40,6 +40,20 @@ export default NextAuth({
       },
     }),
   ],
```

Comment:
This is some janky next-auth stuff, but essentially we're saying with these two callbacks that when we authorize the user, we want to include an id field on the user - this will make the id accessible to us when storing and retrieving form values. This is the issue that is somewhat related to the issue you and Connie were facing, which we can discuss more at standup today, and I can also type up something in the Slack for reference